src/controllers/dlgprefcontroller.cpp

```diff
@@ -388,6 +388,10 @@ void DlgPrefController::slotLoadPreset(int chosenIndex) {
     ControllerPresetPointer pPreset = ControllerPresetFileHandler::loadPreset(
         presetPath, ControllerManager::getPresetPaths(m_pConfig));
 
+    if (!pPreset) {
+        return;
```

Comment:
please add:
 qWarning() << "failed to parse" << presetPath;

Reply:
There's already a qWarning in the XML parser ( src/util/xml.cpp:72)

Reply:
Oh cool.